concourse/pipelines/gpdb_opensource_release.yml

```
@@ -67,23 +67,23 @@ resources:
 - name: python-centos6
   type: gcs
   source:
-    bucket: ((pivotal-gp-internal-artifacts-gcs-bucket))
-    json_key: ((gcs-key))
-    versioned_file: centos6/python-2.7.12.tar.gz
+    json_key: {{concourse-gcs-resources-service-account-key}}
```

```suggestion
    json_key: ((concourse-gcs-resources-service-account-key))
```

The team convention is to use `(())` other than `{{}}`.

---

That's good, There is another story for this issue: https://www.pivotaltracker.com/story/show/168332536

and there are many places needed by fixed in the pipeline config file. we can fix all the problem one time.